api/design/resource.go

```diff
@@ -140,21 +140,22 @@ var _ = Service("resource", func() {
 		})
 	})
 
-	Method("ByKindName", func() {
-		Description("Find resources using name and kind")
+	Method("ByCatalogKindName", func() {
+		Description("Find resources using name of catalog & name and kind of resource")
```

Comment:
Can we change the description to `Find resources using name of catalog, resource name and kind of resource`